### PR TITLE
🧪 Add tests for edge.json.js get handler

### PR DESCRIPTION
**Pull request by jgeofil**
🎯 **What:** The testing gap addressed
Tested the `get` handler exported by `src/pages/edge.json.js`.

📊 **Coverage:** What scenarios are now tested
We verify that the handler correctly returns a `200` status, proper headers including `Content-Type` and `Cache-Control`, and a JSON body containing a valid `time` date string.

✨ **Result:** The improvement in test coverage
100% test coverage for this specific edge endpoint.

### CODE REVIEW

File: src/pages/edge.json.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from "bun:test";
```
[the remark this answers is not in the document]
Comment (jgeofil) [reply src/pages/edge.json.test.js:1]: @gemini @jules apply changes and commit

File: src/pages/edge.json.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { get } from "./edge.json.js";
```
[the remark this answers is not in the document]
Comment (jgeofil) [reply src/pages/edge.json.test.js]: @gemini @jules apply changes and commit
